compiler/luci-interpreter/src/loader/KernelBuilder.cpp

```diff
@@ -16,6 +16,8 @@
 
 #include "loader/KernelBuilder.h"
 
```

Comment:
As I understand, we apply definitions of builders now, but usage(building) of introduced in files implementations you will apply in next PR. If so, LGTM!

Reply:
(optional) Why we make this empty line? I think `KernelBuilder` and `Builders` inclusions could be together =)